wof/core_1_0.py

```diff
@@ -59,19 +59,20 @@ def config_from_file(self, file_name):
         self.default_samplemedium = config.default_samplemedium
 
     '''
-    For WML 1.0 many terms were embedded in the Schemas, which made them not expandable.
-    This provides a standard warning message
+    For WML 1.0 many terms were embedded in the Schemas,
+    which made them not expandable. This provides a standard warning message
     '''
-    def invalid_enum_message(self,name,value, default=None):
-        message = "WaterML 1.0 schema enum issue '{}' not in {}. Use the cuashi_1_1 endpoint ".format(value,name)
+    def invalid_enum_message(self, name, value, default=None):
+        message = ('WaterML 1.0 schema enum issue "{}" not in {}. '
+                   'Use the cuashi_1_1 endpoint ').format(value, name)
         if default is not None:
-            message = "Substituted '"+ default +"'. " + message
+            message = 'Substituted "' + default + '". ' + message
```

Comment:
@ocefpaf Just curious what the advantage of using `'` rather than `"` for strings?